memory-viz/src/memory_model.ts

```diff
@@ -696,21 +775,21 @@ export class MemoryModel {
      * @param {number} y - value for y coordinate of top left corner
      * @param {number} width - the width of the rectangle
      * @param {number} height - the height of the rectangle
-     * @param {object | undefined} style - 1-D object with style properties for a Rough.js object, as per the
+     * @param {Options} style - 1-D object with style properties for a Rough.js object, as per the
      *                        Rough.js API. For instance, {fill: 'blue', stroke: 'red'}.
      */
     drawRect(
         x: number,
         y: number,
         width: number,
         height: number,
-        style?: object
-    ) {
+        style?: Options
+    ): void {
         if (style === undefined) {
             style = this.rect_style;
         }
 
-        style = { ...style, config: this.roughjs_config };
+        style = { ...style, ...this.roughjs_config?.options };
```

Comment:
I'm not sure why you made this change?

Reply:
When I added the type `Options`(exported from roughjs) to `style`, the original line gave an error saying `config` is not a field in `Options`. I checked the `Options` type and realized if `roughjs_config` was typed as `Config`(also from roughjs), we should spread `roughjs_config.options` instead of assigning it to `config`

After making this change, I got an error for the `filepath and a variety of rough-config` test in `cli.spec.tsx`. We were passing `fill=red` in the argument, but the old snapshot had a yellow fill. I believe the issue is related to how we incorrectly used `roughjs_config`

Reply:
Hey @david-yz-liu , after some more testing, these are the results I got when running the following command:
```
npx memory-viz test.json --output="output-new.svg" --roughjs-config seed=12345,fill=green,fillStyle=solid,strokeWidth=20,stroke=red
```

Before the change, this is the output:
![output-old](https://github.com/user-attachments/assets/fed2c214-4934-4598-84df-caa2854f39da)

After the change:
![output-new](https://github.com/user-attachments/assets/cfb43e8a-241b-45a5-bda7-3b688deb7b34)

The reason the old version was working before is because of [line 73](https://github.com/david-yz-liu/memory-viz/blob/master/memory-viz/src/memory_model.ts#L73) in `memory_model.ts`, which hid this bug.

Reply:
Aha, I see that makes sense, thanks for clarifying 👍